detect_secrets/core/usage.py

```diff
@@ -309,6 +309,7 @@ def consolidate_args(args):
 
         active_plugins = {}
```

Comment:
`disabled_plugins` is mainly used to filter off some disabled plugins when reading the plugin list from baseline. Relates to the comment below, it's possible to generate the `disabled_plugins` list on-demand by calculating the difference between all plugins and active plugins. I can add a helper function to the `usage.py`.

I was using `disabled_plugins` earlier to avoid re-iterating on the all plugins to get such list.

Reply:
Gotcha. Makes sense.

Thanks for adding the helper function anyway!